gears/src/commands/client/tx.rs

```diff
@@ -35,10 +34,7 @@ pub struct ClientTxContext {
 }
 
 impl ClientTxContext {
-    pub fn query<
-        Response: Protobuf<Raw> + std::convert::TryFrom<Raw>,
```

Comment:
I removed `Protobuf` bound not only 'cause it's makes no sense for us, but rust don't happy with trait bounds for Error.